src/proposals/mips/mip-b12/MIP-B12.md

```diff
@@ -0,0 +1,91 @@
+# Proposal: Create the xWELL Token
+
+## Summary
+
+This proposal is to create the xWELL token, a token that represents the value of the Well Protocol. The xWELL token will be used to govern the Well Protocol. This is a cross chain token that will go live on both Base and Moonbeam to start. Wormhole relayers will be used in the bridge software to allow for the token to be transferred between chains permissionlessly by end users.
```

Comment:
@ElliotFriedman will this proposal basically allow the Temporal Governor and Governor on Moonbeam to formally take ownership of the xWELL token? Is the desired order of operations to have Halborn complete the audit first, then we'll put this proposal up? Thanks.